tests/testcases/core/EE_Cart_Test.php

```diff
@@ -105,6 +105,7 @@ public function test_add_ticket_to_cart(){
 		$cart = EE_Cart::reset( null, $this->_session );
 		$cart->add_ticket_to_cart( $ticket, $quantity_purchased );
 		$total_line_item = $cart->get_grand_total();
+        // EEH_Line_Item::visualize($total_line_item);
```

Comment:
Debug ?

Reply:
ya but that's in a test file so it can stay imho, tests all get removed prior to release

Reply:
Oh, that's a test file, duh.